kroxylicious-runtime/src/test/java/io/kroxylicious/proxy/internal/filter/FlakyFactory.java

```diff
@@ -32,20 +35,24 @@ public FlakyConfig initialize(FilterFactoryContext context, FlakyConfig config)
         if (config.initializeExceptionMsg() != null) {
             throw new RuntimeException(config.initializeExceptionMsg());
         }
+        INITIALIZED_AND_UNCLOSED += 1;
         return config;
     }
 
     @NonNull
     @Override
     public Filter createFilter(FilterFactoryContext context, FlakyConfig configuration) {
+        Assertions.assertNotNull(configuration);
```

Comment:
Is there a reason to use `Assertions.assertNotNull` here instead of `Objects.requirenonnull` ?